packages/fxa-graphql-api/src/gql/account.resolver.spec.ts

```diff
@@ -83,6 +83,9 @@ describe('#integration - AccountResolver', () => {
         (resolver as any).shouldIncludeLinkedAccounts = jest
           .fn()
           .mockReturnValue(true);
+        (resolver as any).shouldIncludeSecurityEvents = jest
```

Comment:
nit: I see this pattern is already used, but it seems like there should be a better way to mock this without having to cast to any. Unfortunately, I don't know what that is off the top head.

Reply:
Not silly questions, I'll try to answer best I can but not 100% sure. For (1) the front end request all bunch of account related fields and this might just be an easy to include the related data in other tables? There might be a better way to do this, not sure.

For (2), the test will throw errors if it does not return a value.